openc3/lib/openc3/utilities/aws_bucket.rb

```diff
@@ -111,15 +111,20 @@ def list_objects(bucket:, prefix: nil, max_request: 1000, max_total: 100_000)
       token = nil
       result = []
       while true
-        resp = @client.list_objects_v2(bucket: bucket, prefix: prefix, max_keys: max_request)
+        resp = @client.list_objects_v2({
+          bucket: bucket,
+          max_keys: max_request,
+          prefix: prefix,
+          continuation_token: token
+        })
```

Comment:
This was a bug where we weren't passing the continuation token so we only ever got one batch